processor/pom.xml

```diff
@@ -369,6 +395,68 @@
                     </parameter>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
not necessarily needed, just added automatically by intellij when I copied over kotlin example from room. But they are fairly small, so we can also keep them in java